docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -316,6 +317,7 @@ during a successful training session.
 Once the training process completes, and the training process saves the model 
 (denoted by the `Saved Model` message) you can add it to the Unity project and 
 use it with agents having an **Internal** brain type.
+**Note** If you're using Anaconda, you shouldn't just close the Unity Window once the `Saved Model` message appears. Press Ctrl+C in the Anaconda prompt, and this will formally end the training session and export the .bytes file. Otherwise, the .bytes file will never be exported into the ml-agents folder.
```

Comment:
This problem isn't actually Anaconda specific. The problem is a bug, but it is probably worth mentioning for now. Let's use the following language:

**Note**: Do not just close the Unity Window once the `Saved Model` message appears. Either wait for the training process to close the window or press Ctrl+C at the command-line prompt. If you simply close the window manually, the .bytes file containing the trained model is not exported into the ml-agents folder.

Reply:
Ok so I've done that.
Just one question - I was under the impression that the training process never really ends unless it's ended manually. In the docs, it says that it ends then the `Saved Model` message appears, but for me that message never ends the process.
When does it end itself?

Reply:
The length of the training session is determined by the `max_steps` parameter in the `trainer_config.yaml` for the given brain you are training.